taxonomies/ingredients.txt

```diff
@@ -18822,7 +18827,7 @@ de:Baumwollsaatöl, Baumwollsamenöl
 es:aceite de algodón
 fa:روغن پنبه‌دانه
 fi:puuvillansiemenöljy
-fr:huile de coton
+fr:huile de coton, graines de coton
```

Comment:
should be "guile de graines de coton" I think. (and we'll need to add special parsing so that huiles (graines de coton) gets turned into huile de graines de coton.

Reply:
I'll add it under the listing of # huiles in ingredients.pm ?